packages/api/src/controllers/stripe.ts

```diff
@@ -379,6 +453,30 @@ app.post("/migrate-personal-user", async (req, res) => {
           `User ${user.email} has ${subscriptionItems.data.length} subscription items`
         );
 
+        let payAsYouGoItems = [];
+
+        const payAsYouGoPrices = await req.stripe.prices.list({
+          lookup_keys: products["pay_as_you_go_1"].lookupKeys,
+        });
+        payAsYouGoItems = payAsYouGoPrices.data.map((item) => ({
```

Comment:
I think this works? no reason to declare the var early right
```suggestion
        const payAsYouGoPrices = await req.stripe.prices.list({
          lookup_keys: products["pay_as_you_go_1"].lookupKeys,
        });
        const payAsYouGoItems = payAsYouGoPrices.data.map((item) => ({
```